.travis.yml

```diff
@@ -0,0 +1,11 @@
+---
+language: python
```

Comment:
We may need to revisit this when we add node tests. I don't think it matters, but then I'm not sure why it asks...